packages/framework-core/src/booster-register-handler.ts

```diff
@@ -21,16 +23,31 @@ const boosterEventsTypesNames: Array<string> = [
 
 export class RegisterHandler {
   public static async handle(config: BoosterConfig, register: Register): Promise<void> {
-    if (register.eventList.length == 0) {
+    return this.flush(config, register.eventList, register.requestID, register.currentUser)
+  }
+
+  public static async flush(
```

Comment:
I don't fully understand the need of this function, maybe I'm missing something. Let me explain:
- Before there was the `RegisterHandler.handle` method, which basically flushed all the events.
- Why we need another method in this class to do the same? Can't we just make `register.flush` to call the `RegisterHandler.handle` method (through the flusher parameter) and, thus, do not make any change here?